charts/steward/values.yaml

```diff
@@ -0,0 +1,76 @@
+# Default values for steward.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+runController:
+  image:
+    repository: stewardci/stewardci-run-controller
+    tag: 191217_cc244c0
+    pullPolicy: IfNotPresent
+  resources:
+    limits:
+      cpu: 100m
+      memory: 64Mi
+    requests:
+      cpu: 50m
+  podSecurityContext: {}
+    # fsGroup: 2000
+  securityContext: {}
+    # capabilities:
+    #   drop:
+    #   - ALL
+    # readOnlyRootFilesystem: true
+    # runAsNonRoot: true
+    # runAsUser: 1000
+  nodeSelector: {}
+  affinity: {}
+  tolerations: []
+
+tenantController:
+  image:
+    repository: stewardci/stewardci-tenant-controller
+    tag: 191217_cc244c0
+    pullPolicy: IfNotPresent
+  resources:
+    limits:
+      cpu: 50m
+      memory: 32Mi
+    requests:
+      cpu: 10m
+  podSecurityContext: {}
+    # fsGroup: 2000
+  securityContext: {}
+    # capabilities:
+    #   drop:
+    #   - ALL
+    # readOnlyRootFilesystem: true
+    # runAsNonRoot: true
+    # runAsUser: 1000
+  nodeSelector: {}
+  affinity: {}
+  tolerations: []
+
+# imagePullSecrets are used to pull controller images, but no other images.
+imagePullSecrets: []
+
+# pipelineRuns contains common configuration for all pipeline runs
+pipelineRuns:
+  logging:
+    elasticsearch:
+      # indexURL is the URL of the Elasticsearch index to send logs to.
+      # If null or empty, logging to Elasticsearch is disabled.
+      # Example: http://elasticsearch-master.elasticsearch.svc.cluster.local:9200/jenkins-logs/_doc
+      indexURL: ""
+  # resources applies to the Jenkinsfile Runner container.
+  jenkinsfileRunner:
+    image:
+      repository: stewardci/stewardci-jenkinsfile-runner
+      tag: 191216_6d6ef91
+      pullPolicy: IfNotPresent
+    resources:
```

Comment:
see comment above.

Reply:
For the Jenkinsfile Runner container the requested memory is lower than its limit. I'll make them equal.

Reply:
I think memory overcommit makes sense here. Having requests = limits is problematic. If the value is too low, pods may fail with OOM. If the value is too high, we waste resources.

Reply:
The jvm will only see the limits and plan it's memory needs accordingly. If you are afraid of an OOM, there should be a tiny buffer to account for some other processes running in the container.  But overcommiting with a factor of 4 seems risky.

Reply:
We now have 2GiB memory limit/requests. Let's see.